src/RequestBuilder.ts

```diff
@@ -169,6 +173,14 @@ export class RequestBuilder {
 
     return response.data;
   }
+
+  /**
+   * Set the property authorizationHeader
+   * @param authorizationHeader : authorization to set
+   */
+  public setAuthorizationHeader(authorizationHeader: string): void {
+    this.authorizationHeader = authorizationHeader;
+  }
```

Comment:
You could create a setter instead

```suggestion
  public set authorization(authorizationHeader: string) {
    this.authorizationHeader = authorizationHeader;
  }
```